### ISSUE TITLE
Platformio integration, need help

### ISSUE THREAD

**Issue by jtsdana** [michaelbecker/freertos-addons#37]
Hello, I'm trying to add these addons to [platformio](https://platformio.org,) which is a embedded cli for building and dependency management. I already forked this project and tried integrating it, but I need help, as there are some issues that I cannot solve.
https://github.com/DanaDynamics/freertos-addons

**Comment by michaelbecker** [comment]
Hi, what exactly are you having trouble with?

**Comment by jtsdana** [comment]
> Hi, what exactly are you having trouble with?

Hello I needed help with integrating this as a library with Platformio, but had no luck. I was using the STM32F446RE as a target. I also created a thread on a forum https://community.platformio.org/t/add-freertos-wrapper-library-need-help-with-library-json/9942/3 

I have to admit I gave up on it, as I just couldn't get it built without throwing errors, so I have tried to use Mbed instead.
